newsroom/wire/search.py

```diff
@@ -56,20 +66,27 @@ class WireSearchResource(newsroom.Resource):
 def versioncreated_range(created):
     _range = {}
     offset = int(created.get("timezone_offset", "0"))
-    if created.get("created_from"):
+    if created.get("date_from"):
```

Comment:
Keep in mind that if you change the variable name for a filter, then the `My Topic` & `Company Topic` resources will need to be updated, to reflect this change in variable name (so a migration script will be required).

Otherwise these saved topics will not apply the date filters

Reply:
got it !